src/styles/_workspace.scss

```diff
@@ -80,6 +80,14 @@ $code-color-notification: #f9f0d7;
     height: 100%;
     padding-bottom: 0.6rem;
     overflow: auto;
+
+    .fullscreen-button {
+      position: absolute;
+      right: 8px;
+      padding: 5px;
+      cursor: pointer;
+      z-index: 100;
+    }
```

Comment:
I'm not a fan of the use of `position: absolute` unless absolutely necessary, but I guess it's okay here.